QuizApp/LoginController+Animations.swift

```diff
@@ -0,0 +1,86 @@
+import UIKit
+
+extension LoginController {
+    override func viewWillAppear(_ animated: Bool) {
```

Comment:
It's not really a good practice to have life cycle methods (viewDidLoad, viewDidAppear etc.) outside of the original file. Especially not to have a few here and a few there. You could extract this code into a separate method, e.g. prepareForAnimations(), and have that be in this extension and call it from viewWillAppear in the original file.